ui/package.json

```diff
@@ -20,7 +20,7 @@
     "build-storybook": "STORYBOOK=true ember build && build-storybook -s dist",
     "storybook": "STORYBOOK=true start-storybook -p 6006 -s dist",
     "test": "npm-run-all lint test:*",
-    "exam": "ember exam --split=4 --parallel",
+    "exam": "ember exam --split=3 --parallel",
```

Comment:
are you interested in backporting this kinda thing?  we don't gate backport PRs on tests passing, but tests do run when relevant files change (and perhaps someday we should make sure backport tests pass...)